app/konnect/dev-portal/applications/dynamic-client-registration/okta.md

```diff
@@ -93,3 +93,15 @@ From the **My Apps** page in the Dev Portal, follow these instructions:
 After your application has been created, you will see the **Client ID** and **Client Secret**. Please store these values, they will only be shown once. Click **Proceed** to continue to the application's details page.
 
 4. Once your application is created, you will see it in Okta. From your Okta organization select **Applications** from the menu. You will see your application that was created in the Dev Portal, and its corresponding Client ID.
+
+## How to make successful request
+
+In previous steps we obtained the **Client ID** and **Client Secret**. To authorize the request we need to properly attach this client secret pair in the header. Using Insomnia it would look like this.
+
+![Insomnia DCR usage example](/assets/images/docs/konnect/dev-portal/insomnia_dcr_example.png)
```

Comment:
@tomaszwylezek Instead of an image here, could you write up the example request? That way people can use the API product they are familiar with and screen readers can also read the request. 

Would the example be something like `GET example.com/REGISTERED_ROUTE Authorization: Basic CLIENT_ID:CLIENT_SECRET`

Reply:
The reason why I provided the image was that I wanted to direcly show how & where specific data needs to be provided.
With the removal of that image, especially with us having `curl` example some lines below, I don't think it its worth keeping it.
Maybe we can adjust it to something like

```
To authorize the request we need to properly attach this client secret pair in the header. You can do it with any API product or directly use the command line:
```

and then the example of `curl`. What do you think?

Reply:
@tomaszwylezek That makes sense, I think adding the following would work:

"To authorize the request, you must attach this client secret pair in the header. You can do this by using any API product, such as Insomnia, or directly using the command line:"